.travis.yml

```diff
@@ -16,12 +19,14 @@ before_install:
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
 before_script:
-  - npm install -g jpm mozilla-download
+  - npm install -g jpm
   - npm install
+  - export FIREFOX_BIN=`which firefox`
```

Comment:
Do we use the `FIREFOX_BIN` var anywhere (we export the same value below as `JPM_FIREFOX_BINARY`)?